contentful.json

```diff
@@ -57,23 +57,23 @@
     "Client Side Forms": "No",
     "Plugin System": "No",
     "Customizable UI": "Yes",
-    "Custom Field Types": "No",
+    "Custom Field Types": "Yes",
     "User Management": "Yes",
     "Role Based Permissions": "Yes",
     "Document Level Permissions": "No",
-    "OAuth 2.0 Support": "No",
-    "Auditing": "No",
+    "OAuth 2.0 Support": "Yes",
```

Comment:
https://www.contentful.com/developers/docs/references/authentication/#getting-an-oauth-token
https://www.contentful.com/developers/docs/extensibility/oauth/

Reply:
I manually merged the field.